### DISCUSSION TITLE
New install, multiple failures

### DISCUSSION THREAD

**Discussion by wayne47git** [mrlt8/docker-wyze-bridge#238] [Q&A] [answered]
Installed 1.03. It finds my account & all cameras correctly. Still running in "any" mode for the moment.

It starts, I see this error:
```
021/11/19 14:53:35 [27 Lake view] 404 Client Error: Not Found for url: https://wyze-device-alarm-file.s3.us-west-2.amazonaws.com/A4DA2230E9AA/2021-11-   (much more in the URL)
```
The image that the "lake view" camera picks up is from nighttime and never updates. Currently, it's mid afternoon. Both cameras work OK from the app. The "shed cam" shows Unavailable and never gets a thumbnail.

Stream starts, runs a minute and goes down:
```
2021/11/19 14:53:36 [Shed Cam] 🎉 Starting HD 120kB/s Stream for WyzeCam V2 "LAN mode" FW: 4.9.6.199 IP: 192.168.50.103 WiFi: 99%
2021/11/19 14:53:37 [27 Lake view] 🎉 Starting HD 120kB/s Stream for WyzeCam Pan "LAN mode" FW: 4.10.7.1068 🔒 (DTLS) IP: 192.168.0.22 WiFi: 91%
2021/11/19 14:53:40 [27 Lake view] WARNING: Frame not available
2021/11/19 14:53:42 [RTSP][SHED-CAM] ✅ '/shed-cam' stream is UP!
2021/11/19 14:53:42 [RTSP][27-LAKE-VIEW] ✅ '/27-lake-view' stream is UP!
2021/11/19 14:53:48 [2817 Upper] IOTC_ER_DEVICE_OFFLINE
2021/11/19 14:53:48 [2817 Upper] 👻 Camera offline. WILL retry in 20s.
2021/11/19 14:53:48 [Wyze Cam Pan] IOTC_ER_DEVICE_OFFLINE
2021/11/19 14:53:48 [Wyze Cam Pan] 👻 Camera offline. WILL retry in 20s.
2021/11/19 14:53:48 [2817 Lower] IOTC_ER_DEVICE_OFFLINE
2021/11/19 14:53:48 [2817 Lower] 👻 Camera offline. WILL retry in 20s.
2021/11/19 14:54:41 [RTSP][SHED-CAM] ❌ '/shed-cam' stream is down
2021/11/19 14:54:41 [RTSP][27-LAKE-VIEW] ❌ '/27-lake-view' stream is down
```

Nothing else shows up in the logs.

**Comment by wayne47git** [comment]
Follow up.  More issues.

I tried changing to LAN in the Configuration, which seemed to happen when I hit Save. But then the addon tries to grab the thumbnail and hangs:
```
2021/11/20 10:33:50 [Shed Cam] ☁️ Pulling "Shed Cam" thumbnail
```
Since it hangs there, the streams never even start.

So I changed back to ANY, hit Save, accepted the restart. Configuration continued to show LAN. 
Stopping the addon, changing the option, restarting seemed to permit the change to ANY but now it continues to hang on getting the shed cam thumbnail.

It appears that it did, at least once, grab a thumbnail from the shed cam as it has changed. Other camera still has the ancient night shot.

I then set FRESH_DATA, which got the streams to come up again, for 24 seconds.

Running nmap against the hassio box never shows rtsp even when the addon claims that the streams are up.

**Comment by wayne47git** [comment]
And yet more issues. Looks like the process is not being shut down cleanly:
```
🎬 STARTING ALL 6 CAMERAS
2021/11/20 11:36:22 [WyzeBridge] Starting rtsp-simple-server v0.17.8
2021/11/20 11:36:22 [WyzeBridge] WARNING: subprocess 20 is still running
```

So I rebooted the box. Despite "start on boot" being checked, the addon did **not** show that it had started. Log was empty. Starting it manually, once again generated:
```
2021/11/20 11:41:45 [WyzeBridge] Starting rtsp-simple-server v0.17.8
2021/11/20 11:41:45 [WyzeBridge] WARNING: subprocess 20 is still running
```
and a hang at 
```
2021/11/20 11:41:46 [Shed Cam] ☁️ Pulling "Shed Cam" thumbnail
```

**Comment by wayne47git** [comment, marked as answer]
No reponses? Is this project still active?

**Reply by imzenreally** [reply]
https://github.com/mrlt8/docker-wyze-bridge/issues/211

**Reply by wayne47git** [reply]
1.0. got me mostly running. I've got one remaining issue, will create a new discussion.
